styles/stylesheet.css

```diff
@@ -0,0 +1,561 @@
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  max-height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+  background-image: image("images/header-image.jpg");
+  overflow: hidden;
+}
+
+.header-img {
+  width: 100%;
+}
+
+.header-checkout {
+  text-align: left;
+  position: absolute;
+  top: 10%;
+  right: 10%;
+  float: right;
+  clear: both;
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  text-decoration: none;
+  color: #fff;
+}
+
+.header-checkout img {
+  padding-right: 10px;
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: 4px 8px 4px 8px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 10%;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+nav {
+  background-color: #ccc;
+  margin-top: -1rem;
+}
+
+ul {
+  list-style-type: none;
+  padding: 8px 0px 8px 0px;
+  margin-left: 3rem;
+}
+
+li {
+  display: inline;
+  padding-right: 10px;
+}
+
+li a:link {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .8em;
+  text-decoration: none;
+  color: #103861;
+}
+
+li a:visited {
+  color: #000;
+}
+
+li a:hover {
+  color: #2480db;
+}
+
+li a:active {
+  color: #000;
+}
+
+.container-home {
+  margin: 3rem 0 0 3rem;
+}
+
+.container {
+  margin: 3rem 0 0 3rem;
+}
+
+h1 {
+  font-family: 'Merriweather', serif;
+  font-weight: 300;
+  font-style: italic;
+  font-size: 1em;
+  color: #9bc344;
+}
+
+h2 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 1.6em;
+  color: #103861;
+}
+
+h3 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 1.1em;
+  color: #103861;
+}
+
+h4 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 0.7em;
+  color: #103861;
+}
+
+h5 {
+  font-family: 'Merriweather', serif;
+  font-weight: 700;
+  font-size: 0.7em;
+  color: #103861;
+}
+
+p {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 300;
+  font-size: .8em;
+  line-height: 1.2em;
+}
+
+.intro-container {
+  display: block;
+  margin: 1.2rem 0.8rem 4rem 0rem;
+}
+
+.intro-container img {
+  max-width: 100%;
+}
+
+.intro-container h2 p {
+  float: right;
+  padding: 10px 10px 80px 0px;
+}
+
+.dog-profiles {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: start;
+  padding-top: 20px;
+}
+
+.dog-profiles section {
+  border: 1px solid #ccc;
+  width: 25%;
+  margin: 0.8rem 3rem 3rem 0rem;
+}
+
+.dog-profiles img {
+  width: 100%;
+}
+
+button {
+  padding: 0px;
+  border: none;
+}
+
+.dog-profiles a {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  padding: 4px 60px 4px 50px;
```

Comment:
You are using px values for the button sizes, thus they do not scale appropriately different resolutions.